sdk/core/System.Net.ClientModel/src/System.Net.ClientModel.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains building blocks for clients that call cloud services.</Description>
+    <Version>1.0.0-beta.1</Version>
```

Comment:
We already released this version, so will need to rev it prior to releasing any betas.